src/SwedbankPay.Sdk/PaymentInstruments/Swish/ISwishPaymentsResource.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public interface ISwishPaymentsResource
-    {
-        /// <summary>
-        ///     Creates a new swish payment
-        /// </summary>
-        /// <param name="paymentRequest"></param>
-        /// <param name="paymentExpand"></param>
-        /// <returns></returns>
-        Task<SwishPayment> Create(SwishPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
-
-
-        ///// <summary>
-        /////     Gets an existing swish payment.
-        ///// </summary>
-        ///// <param name="id"></param>
-        ///// <param name="paymentExpand"></param>
-        ///// <returns></returns>
-        Task<SwishPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
-    }
-}
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.SwishPayments
+{
+    public interface ISwishResource
+    {
+        /// <summary>
+        ///     Creates a new swish payment
+        /// </summary>
+        /// <param name="paymentRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<ISwishPayment> Create(SwishPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
+
+
+        ///// <summary>
+        /////     Gets an existing swish payment.
+        ///// </summary>
+        ///// <param name="id"></param>
+        ///// <param name="paymentExpand"></param>
+        ///// <returns></returns>
+        Task<ISwishPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)